games-simulation/simutrans/simutrans-nightly.recipe

```diff
@@ -9,18 +9,17 @@ changes which make your savegames unreadable by any official version, or \
 bugs that will destroy your game, or not even run. They are for bug testing \
 only."
 HOMEPAGE="http://www.simutrans.com"
-SOURCE_URI="svn://tron.yamagi.org/simutrans/simutrans/trunk \
-	--username=anon --password="
+SOURCE_URI="svn://servers.simutrans.org/simutrans/trunk"
 SOURCE_FILENAME="simutrans-$portVersion"
 SOURCE_DIR="trunk"
 COPYRIGHT="1997-2004 Hj. Malthaner
-	2005-2015 The Simutrans Team"
+	2005-2017 The Simutrans Team"
 LICENSE="Artistic"
-REVISION="2"
+REVISION="3"
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 PROVIDES="
-	simutrans$secondaryArchSuffix = $portVersion
+	simutrans$secondaryArchSuffix = $portVersion compat >= 120.2
```

Comment:
I can leave it out but this is what I was thinking when I added it:
Not all pakset revisions are compatible with all revisions of simutrans. In this case, a nightly of simutrans (with the version "nightly") that was packaged with this recipe would be slightly more recent but still sufficiently comparable to version 120.2.x of simutrans and therefore compatible with the official paksets for 120.2. As soon as there is a compatibility breakage, the recipe would get bumped to a new revision with a new "compat" entry. The "-nightly" recipes are only intended to be built with haikuporter on one's own for those who want to live on the bleeding edge whereas the versioned recipes could be used to provide stable packages (unfortunately, haikuporter sees "nightly" as a newer revision and would build it if you don't specify the version).
At the moment, the paksets don't check the version of the simutrans package due to the hen/egg problem if you can't use HaikuDepot to install hpkgs (simutrans is only available in haikuports to be built with haikuporter): as soon as you move a pakset hpkg to the packages folder (or double-click on it to use HaikuDepot) you would get a complaint that there is nothing that provides a matching simutrans package so it wouldn't be activated, and if you would then try to move the simutrans package you would see the error message that tells you that there is no matching pakset available (because it wasn't activated before) - you would have to restart Haiku to solve this. This does not happen for packages in the repositories because one would be installed as dependency of the other.